### ISSUE TITLE
aria-label required on the quick create "+" navbar icon

### ISSUE THREAD

**Issue by fenekku** [inveniosoftware/invenio-app-rdm#1907]
From WCAG analysis:

Ensures every ARIA button, link and menuitem has an accessible name

Element Location `.rdm-plus-menu`
Impact: serious (not WCAG AA compliant)

Element Source:

`<div role="menuitem" class="rdm-plus-menu rdm-plus-menu-responsive ui dropdown floating pr-15 computer only" tabindex="0">`

To solve this problem, you need to...

Provide a translated aria-label on that element.

**Comment by fenekku** [comment]
I was thinking "Quick create" ("Quick add" is fine too). It should be with translations: `_("Quick create")` (TBH that's why I didn't tackle it, I haven't had time to familiarize myself with the translation system/procedures for InvenioRDM).

**Comment by fenekku** [comment]
@lhenze are you still looking into this or should we unassign you?

**Comment by lhenze** [comment]
@fenekku apologies, yes, will resume!
